slicer-plugin/NvidiaAIAA/SegmentEditorNvidiaAIAALib/SegmentEditorEffect.py

```diff
@@ -240,6 +242,23 @@ def updateSegmentationMask(self, extreme_points, in_file, modelInfo, overwriteCu
         labelImage = sitk.ReadImage(in_file)
         labelmapVolumeNode = sitkUtils.PushVolumeToSlicer(labelImage, None, className='vtkMRMLLabelMapVolumeNode')
 
+        if cropBox:
```

Comment:
verify the latency... when 1, 2, .... n points are clicked... 
how much it is taking to execute this logic on client side..

Reply:
The run time of this logic should be proportional to the area of bounding box.

We now have two inference pipeline for 3D version of DeepGrow.
The old/existing one is getting the bounding box of all the clicks.
Thus it will become slower as we get more clicks.

The second/new one is just use the latest click and crop a box center around that click.
If we take this approach, then this logic would consume same time for every click.